### PR TITLE
Changelog for 2.1 release

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Waiting for PR #261 to be merged.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-2.1.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Changelog since v2.0.0
+
+## New Features
+
+- Adds support for ListSnapshots secrets ([#252](https://github.com/kubernetes-csi/external-snapshotter/pull/252), [@bells17](https://github.com/bells17))
+
+### Breaking Changes
+
+- Update package path to v2. Vendoring with dep depends on https://github.com/golang/dep/pull/1963 or the workaround described in v2/README.md. ([#240](https://github.com/kubernetes-csi/external-snapshotter/pull/240), [@alex1989hu](https://github.com/alex1989hu))
```
[the remark this answers is not in the document]
Comment (xing-yang) [reply CHANGELOG-2.1.md:10]: Yes, this is a breaking change.

**Comment by xing-yang** [comment]
@yuxiangqian this is ready for review now.

File: CHANGELOG-2.1.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Changelog since v2.0.0
```
Comment (msau42) [inline CHANGELOG-2.1.md:1]: Is it worth splitting up which changes are for the sidecar, and which are for the controller?

Comment (xing-yang) [reply CHANGELOG-2.1.md:1]: sure

Comment (xing-yang) [reply CHANGELOG-2.1.md:1]: Done

**Comment by xing-yang** [comment]
@msau42 @yuxiangqian Addressed your comments.

**Pull request merged**